api/v1beta1/condition_consts.go

```diff
@@ -11,6 +11,10 @@ const (
 	// rolling upgrade for aligning the machines spec to the desired state.
 	EtcdRollingUpdateInProgressReason = "EtcdRollingUpdateInProgress"
 
+	// EtcdMaxNumberOfMachinesReached (Severity=Warning) indicatest that there are 2X replicas while executing a
+	// rolling upgrade for aligning the machines spec to the desired state.
+	EtcdMaxNumberOfMachinesReached = "MaxNumberOfEtcdMachinesReached"
```

Comment:
```suggestion
	```suggestion
	// MaxNumberOfEtcdMachinesReachedReason (Severity=Warning) indicatest that there are 2X replicas while executing a
	// rolling upgrade for aligning the machines spec to the desired state.
	MaxNumberOfEtcdMachinesReachedReason = "MaxNumberOfEtcdMachinesReached"
```
```